packages/tldraw/src/lib/tools/SelectTool/childStates/Idle.ts

```diff
@@ -34,6 +34,28 @@ export class Idle extends StateNode {
 
 	override onPointerMove: TLEventHandlers['onPointerMove'] = () => {
 		updateHoveredId(this.editor)
+
+		const hitShape = this.editor.getShapeAtPoint(this.editor.inputs.currentPagePoint, {
+			hitInside: false,
+			hitLabels: false,
+			margin: HIT_TEST_MARGIN / this.editor.getZoomLevel(),
+			renderingOnly: true,
+		})
+		if (this.isOverArrowLabelTest(hitShape)) {
+			this.editor.updateInstanceState({
+				cursor: {
+					type: 'pointer',
+					rotation: 0,
+				},
+			})
```

Comment:
this is fine, but you could also use `this.editor.setCursor` for these if you wanted